packages/synapse-react-client/src/lib/containers/table/SynapseTable.tsx

```diff
@@ -370,21 +370,22 @@ export default class SynapseTable extends React.Component<
     return (
       <React.Fragment>
         {isExpanded && (
-          <Modal
-            animation={false}
-            show={true}
-            // @ts-ignore
-            onHide={() => this.setState({ isExpanded: false })}
-            dialogClassName={'modal-90w'}
-            backdrop="static"
-          >
-            <Modal.Header
-              // @ts-ignore
-              onHide={() => this.setState({ isExpanded: false })}
-              closeButton={true}
-            ></Modal.Header>
-            <Modal.Body>{content}</Modal.Body>
-          </Modal>
+          <DialogBase
+            open={true}
+            onCancel={() => this.setState({ isExpanded: false })}
+            content={content}
+            title={undefined}
+            maxWidth="xl"
+            fullWidth
+            sx={{
+              '.MuiDialogTitle-root': {
+                padding: 0,
+              },
+              '.MuiDialogContent-root': {
+                border: 'transparent',
+              },
+            }}
```

Comment:
Remove title padding and content dividers, since there is no title. Consider styling adding to DialogBase or to DefaultTheme instead, if this is a common use case